services/harmony/app/models/services/index.ts

```diff
@@ -109,6 +110,10 @@ function validateServiceConfigSteps(config: ServiceConfig<unknown>): void {
           + `Configured to use ${maxBatchInputs}, but will be limited to ${env.maxGranuleLimit}`);
       }
     }
+    if (step.image.match(/harmonyservices\/query\-cmr:.*/) && !step.is_sequential) {
+
+      throw new TypeError(`Invalid is_sequential ${step.is_sequential}. query-cmr steps must always have sequential = true.`);
```

Comment:
nit: should we remove the `is_sequential: true` setting from `query-cmr` step and enforce it in the code? This feels like leaking unnecessary implementation details to the client.

Reply:
This is not a client error. This is an error that shows up when you start Harmony if it is misconfigured. I think I want to keep it this way instead of adding separate handling for query-cmr.

Reply:
I should have said service provider rather than client. The service providers shouldn't have to know the `query-cmr` step is sequential when they set up the service chain in services.yml. The fact that Harmony always execute query cmr sequentially shouldn't have to be spelled out in the service configuration especially when it is something the service providers doesn't care or know. Like I said that I am ok with having it spelled out because we do have other steps that also use the `is_sequential` attribute, but I don't think it is necessary.